docs/Consuming.md

```diff
@@ -1,10 +1,44 @@
 # Consuming messages
 
-Consuming topics can be done by right clicking a topic in the explorer or from the command palette.  Some things to note about consuming:
+Consuming topics can be done:
+
+* with [Kafka Explorer](#kafka-explorer) by right clicking a topic in the explorer.
+* with the [Start command](#start-command) from the command palette.
+
+## Consume with ...
+
+### Kafka explorer
+
+You can start consumer with [Kafka Explorer](Explorer.md#explorer) by right clicking a topic in the explorer:
+
+![Start Consumer with Explorer](assets/start-consumer-from-explorer.png)
+
+Once this command is executed, it creates a consumer group (with generated id) and opens the [Consumer View](#consumer-view) where you can see the producing messages:
+
+![Consumer group / Consumer View](assets/consumer-group-after-starting-from-explorer.png)
+
+In this case, you cannot configure offset, partition and consumer group id. The offset can be just configured with the [kafka.consumers.offset](#kafkaconsumersoffset) preference.

```

Comment:
`In this case, the starting offset can be only be configured via the [kafka.consumers.offset](#kafkaconsumersoffset) preference.`